### DISCUSSION TITLE
White screen on print

### DISCUSSION THREAD

**Discussion by Mario8419** [pdfslick/pdfslick#25] [Q&A]
White screen appears on print in dev and production...

Version: 1.2.7
React, Gatsby

also tested with previous versions

Web example print is also not working on Safari mobile -> Print is opening, but showing only one page to print!

Does anybody else have this problem?
Thx

**Comment by van100j** [comment]
@Mario8419 are you including the `@pdfslick/react/dist/pdf_viewer.css` CSS file as [described in the docs](https://pdfslick.dev/docs/react)? 

There is [printing option in the demo](https://pdfslick.dev/examples/pdf-viewer-app), and it's working.

As for printing on mobile Safari I don't think it works, even with pdf.js. You can try it with the [pdf.js demo viewer app](https://mozilla.github.io/pdf.js/web/viewer.html)

**Reply by Mario8419** [reply]
thx for answer,
yes, added as described...

also added the print container

```
<div id="printContainer" />
    <dialog id="printServiceDialog" className="min-w-[200px]">
      <div className="row">
        <span data-l10n-id="print_progress_message">Preparing document for printing…</span>
      </div>
      <div className="row">
        <progress value="0" max="100"></progress>
        <span data-l10n-id="print_progress_percent" data-l10n-args='{ "progress": 0 }' className="relative-progress">0%</span>
      </div>
      <div className="buttonRow">
        <button id="printCancel" className="dialogButton"><span data-l10n-id="print_progress_close">Cancel</span></button>
      </div>
    </dialog>
```

<img width="936" alt="pdf-print" src="https://github.com/pdfslick/pdfslick/assets/20192321/b680aa1e-a183-4f72-80d0-7300a1a3542c">

**Reply by van100j** [reply]
I really couldn't tell what the issue might be without actually seeing the code. Is this something I can take a look at?
